metrics_utility/automation_controller_billing/package/package_directory.py

```diff
@@ -61,5 +62,9 @@ def ship(self):
         os.makedirs(os.path.dirname(destination_path), exist_ok=True)
         shutil.copyfile(self.tar_path, destination_path)
 
+        # if args contains --verbose, print the destination path
+        if '--verbose' in sys.argv:
+            logger.info(f'tarball saved to: {destination_path}')
```

Comment:
```suggestion
        logger.debug(f'tarball saved to: {destination_path}')
```

logger.debug is only enabled with --verbose .. you don't need to put if --verbose everywhere